src/components/Comments/Comments.jsx

```diff
@@ -0,0 +1,34 @@
+import { useEffect, useState } from "react";
+import { GetAllComments } from "../../apis/apis";
+import { useParams } from "react-router-dom";
+import { Fade } from "react-awesome-reveal";
+
+function Comments() {
+  const { id } = useParams();
+  const [comments, setComments] = useState([]);
+  const [loadingComments, setLoadingComments] = useState(true);
+
+  useEffect(() => {
+    GetAllComments(id).then((res) => {
+      console.log(res);
+      setComments(res.data);
+      setLoadingComments(false);
+    });
+  }, []);
+
+  return (
+    <Fade>
```

Comment:
Would be good to have some text indicating there are no comments (if there are none)
Article 37 has no comments if you'd like to test this out